src/main/resources/static/codegen/built-api-spec.yml

```diff
@@ -2768,40 +2768,46 @@ paths:
         - Task data
       summary: List all tasks
       parameters:
-        - in: query
-          name: type
+        - name: type
+          in: query
```

Comment:
it looks like there's some leftover generation that is missing on `main`, should be in a separate commit/already in `main`

Reply:
looks like it's gone away!